deployments/cardano-prod.nix

```diff
@@ -6,13 +6,24 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["env:production"];
 
     # DEVOPS-64: disable log bursting
     services.journald.rateLimitBurst = 0;
 
     # Initial block is big enough to hold 3 months of transactions
     deployment.ec2.ebsInitialRootDiskSize = mkForce 700;
+
+    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.processConfig = ''
+      init_config:
+
+      instances:
+      - name: cardano-node
+        search_string: ['cardano-node']
+        exact_match: False
+        thresholds:
+          critical: [1, 2]
+    '';
```

Comment:
> We ought to standardize on one strategy.

Moving to DEVOPS-32

> When we're waiting on upstream changes this makes a lot of sense. However, we have to be careful to avoid stomping on each other. In the absence of a single location (per iohk-nixops branch) for setting the NIX_PATH, when there are multiple pending upstream changes we risk accidentally introducing regression.

As long as we wait for upstream and bump nixpkgs at correct locations, it shouldn't be an issue (once we have it documented ofc).

> Would it help to make an official IOHK fork of nixpkgs and coordinate changes there? Ultimately we benefit from maintaining parity with nixos/nixpkgs, so I'm not suggesting we diverge any more than necessary.

I'd like to avoid this if possible. Upstream moves fast and I'm among top 10 contributors, so I can provide valuable feedback on PRs from within IOHK. If we are in very fast hurry we can just pin a fork branch.

I'd rather wait a few days and make sure upstream gets feedback than rely on a forked branch that will do the job.

This might change in the future if it becomes unbearable, but I don't expect much trouble.